.github/ISSUE_TEMPLATE/article-needs-fixing

```diff
@@ -0,0 +1,23 @@
+---
+name: 'Bug Report: Issues with \news or guide articles or documentation'
+about: Report issue with a specific article, like broken links, typos, missing parts, etc
+title: ''
+labels: 'status: needs help for triage, platform: news, platform: forum'
+assignees: ''
+
+---
+
+<!-- 
+NOTE: If you want to become an author of freecodecamp, you can find everything here: https://www.freecodecamp.org/news/developer-news-style-guide/
+-->
+
+**Describe your problem and how to reproduce it:**
+
+
+**Add a Link to the page with the problem:**
+
+
+**How would you update it?**
+
+
+**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**
```

Comment:
```suggestion
**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**

```
Newline

Reply:
I have included the suggested changes, sorry for the double commit, github is confusing